plugins/pulp_rpm/plugins/distributors/yum/metadata/updateinfo.py

```diff
@@ -144,7 +144,12 @@ def add_unit_metadata(self, item):
         else:
             repo_unit_nevra = None
 
+        seen_pkglists = []
```

Comment:
Since your primary use case for this structure is to test membership, a ``set`` could be a better fit. You'll have to cast ``packages`` as a tuple, but that's a fine idea anyway.